src/cljs/ataru/hakija/hakija_readonly.cljs

```diff
@@ -26,21 +26,28 @@
        (contains? field-descriptor :koodisto-source)))
 
 (defn text [field-descriptor application lang]
-  (let [answer ((answer-key field-descriptor) (:answers application))]
+  (let [answer            ((answer-key field-descriptor) (:answers application))
+        multi-values->:li (partial map-indexed (fn [idx value]
```

Comment:
Extract from let-block?

Reply:
Discussed this face-to-face. Decided to leave this as-is for now.